### PR TITLE
Travis passing on cqm integration

### PR DESCRIPTION
**Pull request by lukoktonos**
The goal of this PR is to get travis green again so that breaking changes don't go unnoticed while developing on the feature branch.

codecov will need to be ignored for this PR but then should be green for subsequent PRs into feature branch

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-2021
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable) (cant run frontend tests locally due to chrome version issue)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) ) N/A
- [x] Code coverage has not gone down and all code touched or added is covered. (went down and adjusted thresholds to get travis passing)
- [x] Automated regression test(s) pass N/A

**Reviewer 1:**

Name: @zlister 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Review by zlister — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ script:
   - bundle exec rake teaspoon USE_COVERAGE=default
   - codecov
   - bundle exec brakeman -qAzw1
-  - bundle exec bundle-audit check --update
+  # - bundle exec bundle-audit check --update
```
Comment (jbradl11) [inline .travis.yml:31]: I'd recommend making a Jira task to revert this change once we are ready. Just in case it gets forgotten about. In that jira task, list the other things we should remember to "Revert"

File: lib/tasks/cql_testing.rake [hunk 2]
```diff
@@ -34,6 +34,28 @@ namespace :bonnie do
       fixture_exporter.export_records(record_file_path)
     end
 
+    desc %{Export backend cqm fixtures for a given user account
```
Comment (jbradl11) [inline lib/tasks/cql_testing.rake:37]: Is this a new rake task? Or is the diff being weird?

File: .travis.yml (hunk 1 above)
Comment (lukoktonos) [reply .travis.yml:31]: Good idea, made https://jira.mitre.org/browse/BONNIE-2022

File: lib/tasks/cql_testing.rake (hunk 2 above)
Comment (lukoktonos) [reply lib/tasks/cql_testing.rake:37]: This is a new rake task I added.  I'm not sure if there was a non-committed rake task that properly exported to the current backend directory structure or not, but I didn't see one so I had to make this.

This rake task run with bonnie-fixtures should be the source of truth for backend fixtures ideally.  Any one-off measures should be added to bonnie-fixtures and then exported from there, IMO

Comment (jbradl11) [reply lib/tasks/cql_testing.rake:37]: @hackrm Did you make a similar rake task?

**Review by jbradl11 — APPROVED** [review]
